<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>631</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>数值的输入与输出</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_count">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>单价：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prince">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_qDebug">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>qDebug测试</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_sum">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>110</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>计算总价</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>80</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>总价：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sum">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>70</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>进制转换</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>十进制</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numten">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_other1">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton_other1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>二进制</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numtwo">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_other2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton_other2</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
